Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -137,21 +140,32 @@ public struct WorldScaleGeoTrackingSceneView: View {
         .task {
             do {
                 try await locationDataSource.start()
-                await withTaskGroup(of: Void.self) { group in
-                    group.addTask {
-                        for await location in locationDataSource.locations {
-                            lastLocationTimestamp = location.timestamp
-                            currentLocation = location
-                            await updateSceneView(for: location)
-                        }
-                    }
+                for await location in locationDataSource.locations {
+                    lastLocationTimestamp = location.timestamp
+                    currentLocation = location
+                    updateSceneView(for: location)
                 }
             } catch {}
         }
-        .toolbar {
-            ToolbarItem(placement: .bottomBar) {
-                if !calibrationViewIsHidden {
-                    calibrationView
+        .overlay(alignment: calibrationViewAlignment) {
+            if configuration is ARWorldTrackingConfiguration,
```

Comment:
You can leave this condition here. I figured that it may be still worth having the calibration view for geo-tracking even if the adjustments are minor. I'll tweak the conditions in #574